src/blocks/pacman.rs

```diff
@@ -142,14 +142,13 @@ impl ConfigBlock for Pacman {
         Ok(Pacman {
             id,
             update_interval: block_config.interval,
-            format: FormatTemplate::from_string(&block_config.format)
+            format: FormatTemplate::new(&block_config.format, None)
                 .block_error("pacman", "Invalid format specified for pacman::format")?,
-            format_singular: FormatTemplate::from_string(&block_config.format_singular)
-                .block_error(
-                    "pacman",
-                    "Invalid format specified for pacman::format_singular",
-                )?,
-            format_up_to_date: FormatTemplate::from_string(&block_config.format_up_to_date)
+            format_singular: FormatTemplate::new(&block_config.format_singular, None).block_error(
+                "pacman",
+                "Invalid format specified for pacman::format_singular",
+            )?,
```

Comment:
Off topic but you can remove those `.block_error()`s to be consistent with other blocks.